pam/pam-module-transaction-dummy_test.go

```diff
@@ -0,0 +1,218 @@
+package main
```

Comment:
Same comment about the package name and the file name...
IMHO, this should be `package pam` or, even better, `package pam_test`  (this way you would be able to see how the API looks for outside packages).
And file name should be something like `module-transaction-mock_test.go`

Reply:
Ok, done... I've kept the `dummy` suffix for now because what I said on mocks, but I'm happy to change it if that's not convincing :)